apps/ejabberd/src/mongoose_acc.erl

```diff
@@ -64,10 +64,9 @@ from_kv(K, V) ->
 -spec from_element(xmlel()) -> t().
 from_element(El) ->
     #xmlel{name = Name, attrs = Attrs, children = Children} = El,
-    Type = xml:get_attr_s(<<"type">>, Attrs),
-    {Xmlns, Cmd} = read_child(Children),
-    #{element => El, mongoose_acc => true, name => Name, attrs => Attrs, type => Type,
-      xmlns => Xmlns, command => Cmd}.
+    Type = exml_query:attr(El, <<"type">>, undefined),
+    Acc = #{element => El, mongoose_acc => true, name => Name, attrs => Attrs, type => Type},
+    read_children(Acc, Children).
```

Comment:
I'm reluctant to inspect every stanza so deeply. When exactly do we need to read the children? Can we limit it to some small number of places and do it explicitly only for stanza we know we'll need this?